ext/hosting.py

```diff
@@ -120,23 +122,26 @@ async def gamemodes(self, ctx, mode):
     @commands.command(name="end")
     @commands.has_any_role(702601007368241173, 702604059613462589, 702604111450996818, 702605281204502638)
     async def end(self, ctx):
-        toscd = self.bot.get_guild(702600628601356359)
+        guild = self.bot.get_guild(702600628601356359)
         await ctx.message.delete()
-        await toscd.get_channel(702602469812863106).set_permissions(toscd.get_role(702694502204178554), send_messages=False)
+        await guild.get_channel(702602469812863106).set_permissions(guild.get_role(702694502204178554), send_messages=False)
         await ctx.send('This gamenight has ended, go to <#702602837737078897> if you wish to continue playing.')
 
     @commands.command(name="start")
     @commands.has_any_role(702601007368241173, 702604059613462589, 702604111450996818, 702605281204502638)
-    async def start(self, ctx):
-        toscd = self.bot.get_guild(702600628601356359)
+    async def start(self, ctx, cohost: typing.Optional[discord.User]):
+        guild = self.bot.get_guild(702600628601356359)
         await ctx.message.delete()
-        await toscd.get_channel(702602469812863106).set_permissions(toscd.get_role(702694502204178554), send_messages=True)
-        await ctx.send(f'Gamenight opened! ({ctx.author.mention})')
+        await guild.get_channel(702602469812863106).set_permissions(guild.get_role(702694502204178554), send_messages=True)
+        if cohost == None:
+            return await ctx.send(f'Gamenight opened! ({ctx.author.mention})')
+        else:
+            return await ctx.send(f'Gamenight opened! ({ctx.author.mention}, {cohost.mention})')
 
     @commands.command(name="takeover")
     @commands.has_any_role(702601007368241173, 702604059613462589, 702604111450996818, 702605281204502638)
     async def takeover(self, ctx):
-        toscd = self.bot.get_guild(702600628601356359)
+        guild = self.bot.get_guild(702600628601356359)
```

Comment:
Codacy found an issue: [Unused variable 'guild'](https://app.codacy.com/manual/RainbowLegend/aerobot/pullRequest?prid=5767464)

Reply:
it actually is being used smh codacy